InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py

```diff
@@ -103,12 +108,14 @@ def setup(self) -> None:
 
```

Comment:
This cuda() could be the reason why the multi-gpu option is not working - discussed in more detail at voice

Reply:
About .eval(). looking at what happens inside the forward pass of deepmil I don't think we should set it to eval mode if we're in the fine-tune case

Reply:
This is done, thanks